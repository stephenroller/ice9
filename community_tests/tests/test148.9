# forward & proc have diff names							succeed

forward foo(a, b : int, c : bool, d : str)

proc foo(e, f : int, g : bool, h : str)
	write e;
	write f;
	write ?g;
	write h;
end

foo(0, 1, true, "Hi");