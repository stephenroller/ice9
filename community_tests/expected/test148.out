Loading file: generated/test148.tm
0 
1 
1 
Hi
Number of instructions executed = 69
